frontend/src/components/admin/StoryTranslationsTable.tsx

```diff
@@ -138,6 +145,16 @@ const StoryTranslationsTable = ({
           </Link>
         </Td>
         <Td>{lastEditedDate(storyTranslationObj)?.toLocaleDateString?.()}</Td>
+        <Td>
+          <IconButton
+            aria-label={`Export story translation ${storyTranslationObj?.storyTranslationId}`}
+            background="transparent"
+            icon={<Icon as={MdOutlineFileDownload} />}
+            width="fit-content"
+            isDisabled={storyTranslationObj.stage !== "PUBLISH"}
```

Comment:
![image](https://user-images.githubusercontent.com/15113249/147768837-3bf7ae04-12e4-4ee0-8ede-5281f02a7f3f.png)
can you add in this tooltip when the button is disabled